web_poet/__init__.py

```diff
@@ -1,6 +1,19 @@
 from .pages import WebPage, ItemPage, ItemWebPage, Injectable
-from .page_inputs import HttpResponse, HttpResponseBody, HttpResponseHeaders
+from .requests import (
+    request_backend_var,
+    HttpClient,
```

Comment:
what's interesting is that HttpClient also can be a PO dependency :) I wonder if we should part from having page_inputs module, and just split it. E.g. web_poet.http, or something like this.

Reply:
This is a good thought to ponder. I think we have 3 main choices for this which are:

1. Move `HttpClient` from **requests.py** to **page_inputs.py**
2. Create a **page_inputs/** subpackage which houses **page_inputs/meta.py** and **page_inputs/http.py** (`HttpClient` is here)
3. Disband the **page_inputs.py** and simply have modules like **meta.py** and **http.py** (`HttpClient` is here)

I'm torn between 1 and 2 since having some sort of subpackage/module named **page_inputs** helps portray the idea that anything within it could be a PO dependency.

What do you think? Perhaps there might be other good options aside from these that I might've missed?

Reply:
I'm also not sure :) Let's handle it separately.

Reply:
Sure. I've filed a draft in https://github.com/scrapinghub/web-poet/pull/37 which explores **option 2** to see how it may look like.